readme.md

```diff
@@ -102,27 +109,24 @@ Show the system `Services` submenu when right-clicking text on macOS.
 
 Note: Due to [a bug in the Electron implementation](https://github.com/electron/electron/issues/18476), this menu is not identical to the "Services" submenu in the context menus of native apps. Instead, it looks the same as the "Services" menu in the main App Menu. For this reason, it is currently disabled by default.
 
-#### showLookUpSelection
-
-Type: `boolean`<br>
-Default: `true`
-
-Show the `Look Up [selection]` menu item when right-clicking text on macOS.
-
 #### labels
 
 Type: `Object`<br>
 Default: `{}`
 
-Overwrite labels for the default menu items. Useful for i18n.
+Override labels for the default menu items. Useful for i18n.
+
+The placehlder `{selection}` may be used in any label, and will be replaced by the currently selected text.
```

Comment:
This needs to be more comprehensive. It needs to mention what the user has to check for and how to guard against selection not being available (like we do in the built-in menu. Maybe show an example of a custom menu using `{selection}`?
```suggestion
The placeholder `{selection}` may be used in any label, and will be replaced by the currently selected text.
```